src/pages/auth/login/Login.css

```diff
@@ -0,0 +1,44 @@
+/* ------------------------- || LOGIN MAIN ----------------------- */
+.login-main {
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+  justify-content: space-around;
+  min-height: 85vh;
+  background-color: var(--component-bg-color);
+}
+
+/* ------------------ || FORM CONTAINER ------------------ */
+.form-container {
+  padding: 1.5rem 3rem;
+  align-items: center;
+}
+.login-header,
+.form-btn {
+  margin-bottom: 1.5rem;
+}
+.form-btn {
+  background-color: var(--primary-color);
+  color: var(--nav-title-color);
+}
+.login-header,
+.signup-link {
+  text-align: center;
+}
+.form-invalid-message {
+  font-size: small;
```

Comment:
If possible you can give a font size value to it that fits with this criteria!!

Reply:
ideally, this is also one of the value we can give in css. not necessarily `px` or `rem` is needed.